package/config/configuration/ampathforms/poc_covid19.json

```diff
@@ -0,0 +1,195 @@
+{
+	"name": "POC Covid",
+	"description" : "Test 1 Description",
+    "version" : "1",
```

Comment:
Seems there's some weird spacing going on. Is it possible to run this through a code formatter like Prettier?

Reply:
Done